hugo/content/overview.md

```diff
@@ -17,13 +17,13 @@ NGINX Agent is a companion daemon for your NGINX Open Source or NGINX Plus insta
 - Notifications of NGINX events
 
 
-{{< img src="docs/grafana-dashboard-example.png" caption="Grafana  dashboard showing metrics reported by NGINX Agent" alt="Grafana dashboard showing metrics reported by NGINX Agent" width="99%">}}
+{{< img src="grafana-dashboard-example.png" caption="Grafana  dashboard showing metrics reported by NGINX Agent" alt="Grafana dashboard showing metrics reported by NGINX Agent" width="99%">}}
 
 ## How it Works
 
 NGINX Agent runs as a companion process on a system running NGINX. It provides gRPC and REST interfaces for configuration management and metrics collection from the NGINX process and operating system. NGINX Agent enables remote interaction with NGINX using common Linux tools and unlocks the ability to build sophisticated monitoring and control systems that can manage large collections of NGINX instances.
 
-{{< img src="docs/agent-flow.png" caption="How Agent works" alt="How NGINX Agent works" width="99%">}}
+{{< img src="agent-flow.png" caption="How Agent works" alt="How NGINX Agent works" width="99%">}}
 
```

Comment:
The agent-flow.png should be moved as part of this?

Reply:
agent-flow.png used by hugo docs has moved with the hugo docs to the hugo\static folder
 
  agent-flow.png used by the README.md in the root of the repository stays in the \docs folder. This file is not related to the hugo documentation